### PR TITLE
Feature/5949 6146 update exposure submission intro view controller

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
This PR covers the task layout and wording changes to the exposure submission intro view controller.
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6146

JIRA Story
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5649

IMG_0519

### CODE REVIEW

**Review by BLUM78691 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift [hunk 1]
```diff
@@ -30,20 +30,13 @@ class ExposureSubmissionIntroViewModel {
...
-					UIImage(named: "Illu_Submission_Funktion1"),
+					UIImage(named: "Illu_Test_registration"),
 					accessibilityLabel: AppStrings.ExposureSubmissionDispatch.accImageDescription,
 					accessibilityIdentifier: AccessibilityIdentifiers.General.image,
 					height: 200
 				),
 				separators: .none,
-				cells: [
-					.body(
-						text: AppStrings.ExposureSubmissionDispatch.description,
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift:41]: Same here: Are the String and AccessibilityIdentifier still used by someone else? If not, we could remove them...

Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift:41]: The AccessibilityIdentifiers are not used anywhere else. Didn't find any UITest using this.

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift [hunk 2]
```diff
@@ -30,20 +30,13 @@ class ExposureSubmissionIntroViewModel {
 		return DynamicTableViewModel.with {
 			$0.add(.section(
 				header: .image(
-					UIImage(named: "Illu_Submission_Funktion1"),
+					UIImage(named: "Illu_Test_registration"),
```
[the remark this answers is not in the document]
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift:33]: Good point. Indeed isn't used somewhere else. I removed it.

**Review by pascal-brause — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Pull request merged**